scripts/docker/start.sh

```diff
@@ -10,6 +10,13 @@ fi
 if [ "$RAILS_ENV" == "development" ]; then
   exec foreman start
 else
+  USER_DATA=$(curl --fail http://169.254.169.254/latest/user-data || echo "")
+
+  if [ "$USER_DATA" == "EMERGENCY_MODE" ]
+  then
+    git pull
```

Comment:
`git pull` should be fine since there is only 1 origin from the docker build `git clone` and it allows the branch that docker built off to be pulled from.